docs/source/conf.py

```diff
@@ -110,7 +110,15 @@
 html_logo = "_static/delta-lake-white.png"
 html_favicon = "_static/favicon.ico"
 
-html_theme_options = {}
+html_theme_options = {
+    "show_toc_level": 2,
```

Comment:
what is this for? isnt this supposed to show 2 levels always expanded? is it not working?

Reply:
removed this block.